src/env.js

```diff
@@ -47,6 +47,9 @@ const vector = {
 };
 let config = {
     environment: "production",
+    checkForUpdatesAtBoot: false,
+    checkForUpdatesInterval: 1000 * 3600 * 24 * 5,  // Check every 5 days
+    checkForUpdates: false,
```

Comment:
`checkForUpdatesAtBoot` is already referenced at the bottom of this config block. These should go there.

Reply:
ok